updates.md

```diff
@@ -1,3 +1,47 @@
 Sample updates
 
 Test link markup
+
+Some other comment
+
+Sample updates
+
+Test link markup
+
+Some other comment
+
+Sample updates
+
+Test link markup
+
+Some other comment
+
```

Comment:
this is cool too:

```
def something
```

something: https://github.com/mhahn/devel-github-integration/pull/2/files